src/MoBi.Core/Commands/AddPathAndValueEntityToBuildingBlockCommand.cs

```diff
@@ -65,7 +65,7 @@ public AddParameterValueToBuildingBlockCommand(PathAndValueEntityBuildingBlock<P
 
    public class AddInitialConditionToBuildingBlockCommand : AddPathAndValueEntityToBuildingBlockCommand<InitialCondition>
    {
-      public AddInitialConditionToBuildingBlockCommand(PathAndValueEntityBuildingBlock<InitialCondition> initialConditionsBuildingBlock, InitialCondition pathAndValueEntity)
+      public AddInitialConditionToBuildingBlockCommand(IBuildingBlock<InitialCondition> initialConditionsBuildingBlock, InitialCondition pathAndValueEntity)
```

Comment:
This is what saved a bunch of refactoring of commands and tasks. In a lot of places we didn't need the specification of ```PathAndValueEntityBuildingBlock<InitialCondition>```, we really just needed ```IBuildingBlock<InitialCondition>```